NullReferenceException on RoomUnloading

Good Afternoon,

Slowly working through errors and submitting pull requests of fixes, this seems to be the main error atm.
This occurs when a user does :unload or when the room is unloaded due to no users being in it.

```13:07:58 - [ERROR] Exception:
System.NullReferenceException: Object reference not set to an instance of an object
  at Plus.HabboHotel.Rooms.Room.Dispose () [0x0005c] in /Users/josh/Habbo Related/PlusEMU-R3/HabboHotel/Rooms/Room.cs:754 
  at Plus.HabboHotel.Rooms.RoomManager.UnloadRoom (System.Int32 roomId) [0x00014] in /Users/josh/Habbo Related/PlusEMU-R3/HabboHotel/Rooms/RoomManager.cs:162 
  at Plus.HabboHotel.Rooms.Room.ProcessRoom () [0x0009e] in /Users/josh/Habbo Related/PlusEMU-R3/HabboHotel/Rooms/Room.cs:452 ```

--- a/HabboHotel/Rooms/Room.cs
+++ b/HabboHotel/Rooms/Room.cs
@@ -42,7 +42,6 @@ namespace Plus.HabboHotel.Rooms
         public DateTime lastRegeneration;
 
         public Task ProcessTask;
-        public ArrayList ActiveTrades;
 
         public TonerData TonerData;
         public MoodlightData MoodlightData;
@@ -750,10 +749,6 @@ namespace Plus.HabboHotel.Rooms
                         ProcessTask.Dispose();
                 }
                 catch { }
-                
-                if (ActiveTrades != null)
-                    if (ActiveTrades.Count > 0)
-                        ActiveTrades.Clear();
 
                 TonerData = null;
                 MoodlightData = null;

--- a/HabboHotel/Rooms/Room.cs
+++ b/HabboHotel/Rooms/Room.cs
@@ -750,9 +750,10 @@ namespace Plus.HabboHotel.Rooms
                         ProcessTask.Dispose();
                 }
                 catch { }
-
-                if (ActiveTrades.Count > 0)
-                    ActiveTrades.Clear();
+                
+                if (ActiveTrades != null)
+                    if (ActiveTrades.Count > 0)
+                        ActiveTrades.Clear();
 
                 TonerData = null;
                 MoodlightData = null;

